demos/oms-all-deploy/nestedtemplates/omsAutomation.json

```diff
@@ -54,54 +53,21 @@
       "metadata": {
         "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
       }
+    },
+    "psModules": {
+      "type": "object",
+      "metadata": {
+        "description": "URIs of the PS modules"
+      }
     }
   },
   "variables": {
-    "psModules": {
```

Comment:
revert this change - don't worry about the BP warning

Reply:
reverted